SeaBattle/Sea.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+using System.Security.Cryptography;
+using System.Xml.Linq;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        public List<BaseShip> Ships { get; set; }
+        public int SeaWidth;
+        public int SeaHeight;
+
+        // Property to provide indexer
+        public BaseShip this[int i]
+        {
+            get {
+                if (i > 0 && i < Ships.Count)

```

Comment:
What if I want to get the ship with an index 0?

Reply:
It must be i >= 0, because you can do Sea[0], fixed.

Reply:
Same, don't see the change.

Reply:
Fixed now.